### ISSUE TITLE
Collapsible-fields not accessible by keyboard

### ISSUE THREAD

**Issue by buddh4** [humhub/humhub#4945]

**Comment by luke-** [comment]
![image](https://user-images.githubusercontent.com/4736168/137012946-89bdf326-bee2-470e-a343-3970970a80cf.png)

**Comment by Soundwave2142** [comment]
@luke Done. Problem is with custom check-boxes itself, had to do some css fixes and rebuild theme.css. Let me know if it works now as you wanted.
